src/tools/converters.ts

```diff
@@ -0,0 +1,3 @@
+export const convertNsToMs = (ns: number): number => {
```

Comment:
Nitpick, siubjective, feel free to ignore, I think it is more readable this way :
```suggestion
export const nanosecondsToMilliseconds = (ns: number): number => {
```